packages/libhdf5/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: libhdf5
+  version: 1.12.1
+
+source:
+  sha256: e6dde173c2d243551922d23a0387a79961205b018502e6a742acb30b61bc2d5f
+  url: https://github.com/HDFGroup/hdf5/archive/refs/tags/hdf5-1_12_1.tar.gz
+
+requirements:
+  run:
+    - zlib
+
+build:
+  library: true
+  script: |
+    mkdir -p build;
+    cd build \
+        && LDFLAGS="-s NODERAWFS=1" emcmake cmake ../ \
+        -DCMAKE_CROSSCOMPILING_EMULATOR="$PYODIDE_ROOT/emsdk/emsdk/node/14.18.2_64bit/bin/node" \
```

Comment:
Would it be easier to maintain if we wrote this in a version agnostic way? :)
Maybe with `$(which node)` or a variable? :)

Reply:
Of course - that would be great!

Reply:
`$(which node)` isn't necessarily a good idea though because you want to use the emscripten node, and I'm concerned that it would find a different node in a lot of cases.

Reply:
If email/ only contains one version at a time, we could also rely on a bash expansion and use * for it